central/processlisteningonport/datastore/datastore_impl.go

```diff
@@ -445,11 +455,11 @@ func getProcessUniqueKeyFromParts(containerName string,
 
 func getPlopProcessUniqueKey(plop *storage.ProcessListeningOnPortFromSensor) string {
 	return getProcessUniqueKeyFromParts(
-		plop.Process.ContainerName,
-		plop.Process.PodId,
-		plop.Process.ProcessName,
-		plop.Process.ProcessArgs,
-		plop.Process.ProcessExecFilePath,
+		plop.GetProcess().GetContainerName(),
+		plop.GetProcess().GetPodId(),
+		plop.GetProcess().GetProcessName(),
+		plop.GetProcess().GetProcessArgs(),
+		plop.GetProcess().GetProcessExecFilePath(),
```

Comment:
I am not sure if this is necessary, but the change is consistent with getProcessUniqueKey.